backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -259,6 +259,30 @@ class TestOperator extends WholeStageTransformerSuite {
     checkLengthAndPlan(df, 1)
   }
 
+  test("Test round function") {
+    val df = runQueryAndCompare("SELECT round(cast(l_orderkey as int), 2)" +
+      "from lineitem limit 1") { checkOperatorMatch[ProjectExecTransformer] }
+    df.show()
+    df.explain(false)
+    df.printSchema()
+  }
+
+  test("Test greatest function") {
+    val df = runQueryAndCompare("SELECT greatest(l_orderkey, l_orderkey)" +
+      "from lineitem limit 1" ) { checkOperatorMatch[ProjectExecTransformer] }
+    df.show()
+    df.explain(false)
+    df.printSchema()
+  }
+
+  test("Test least function") {
+    val df = runQueryAndCompare("SELECT least(l_orderkey, l_orderkey)" +
+      "from lineitem limit 1" ) { checkOperatorMatch[ProjectExecTransformer] }
+    df.show()
+    df.explain(false)
+    df.printSchema()
+  }
+
```

Comment:
Could you increase the limit to verify more test data, such as 1000?

Reply:
Because the l_orderkey is growing naturally.  If  limit 1000, the l_orderkey  is also between 0 ~1000. Just validate one case.

Reply:
Also, I am designing new validate Suite, independent TPCH items.  Try to cover more cases.

Will submit in following functions support PR.